unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UIRadialRainbow : MonoBehaviour
+{
+    Material _targetMat;
+    public Image img;
+    public Texture mask;
+    public Color innerColor = Color.white;
+    public float frameThickness = 1f;
+    public bool useTexture;
+    public Texture gradientTexture;
+    public Gradient gradient;
+    public Vector2 speed;
+    [Range(-360, 360)]
+    public float rotation;
+    void Start()
```

Comment:
In order to have a mode readable code, lets put an space here, between the variables block and the header of the next function:

```suggestion
    public float rotation;
    
    void Start()
```